crates/bevy_pbr/src/deferred/mod.rs

```diff
@@ -462,11 +460,19 @@ pub fn prepare_deferred_lighting_pipelines(
         dither,
         shadow_filter_method,
         (ssao, ssr, distance_fog),
-        (normal_prepass, depth_prepass, motion_vector_prepass),
+        (normal_prepass, depth_prepass, motion_vector_prepass, deferred_prepass),
         has_environment_maps,
         has_irradiance_volumes,
     ) in &views
     {
+        // If there is no deferred prepass, remove the old pipeline if there was
```

Comment:
I wonder if there's the same bug in meshlets... Feel free to ignore that for this PR though, the entire CPU-side of meshlets needs redoing once the GPU side is finished.